common/path.c

```diff
@@ -109,10 +109,12 @@ expand_homedir (const char *remainder)
 {
 	const char *env;
 
+#ifdef OS_UNIX
```

Comment:
Rather than adding `#ifdef`, I would suggest adding a stub `getauxval` in common/compat.c.